specs/DesignDiscussions/InputLabels.md

```diff
@@ -91,6 +129,42 @@ If we want to allow the host to control formatting of the labels, either in gene
 ```
 Another option is to not put this in host config, and instead control this styling via support of native styling on required and optional labels (i.e. CSS, Xaml Styles, etc). This decision speaks to a larger discussion of the direction of host config that is beyond the scope of this document.
 
+> ### Host side styling
```

Comment:
>### [](start = 2, length = 3)

This is the 2nd of the 2 options mentioned in the Host Formatting section yes? If so, this should be a subhead at the same level as TextFormatting (####) - though ideally use numbered values to indicate numbered options for clarity/ease of discussion. so 1. and 2. in addition to or inlieu of sub-heads.